src/client/app/utils/api/UploadCSVApi.ts

```diff
@@ -4,53 +4,75 @@
-			refreshHourlyReadings: uploadPreferences.refreshHourlyReadings ? BooleanTypes.true : BooleanTypes.false,
-			refreshReadings: uploadPreferences.refreshReadings ? BooleanTypes.true : BooleanTypes.false,
-			honorDst: uploadPreferences.honorDst ? BooleanTypes.true : BooleanTypes.false,
-			relaxedParsing: uploadPreferences.relaxedParsing ? BooleanTypes.true : BooleanTypes.false,
-			useMeterZone: uploadPreferences.useMeterZone ? BooleanTypes.true : BooleanTypes.false
-		};
-		for (const [preference, value] of Object.entries(uploadPreferencesForm)) {
-			formData.append(preference, value.toString());
-		}
-		formData.append('csvfile', readingsFile); // It is important for the server that the file is attached last.
-		return await this.backend.doPostRequest<void>('/api/csv/readings', formData);
+export const submitMeters = async (uploadPreferences: MetersCSVUploadPreferencesItem, metersFile: File,
+	dispatch: Dispatch): Promise<apiResponse> => {
+	const backend = new ApiBackend();
+	const formData = new FormData();
+	// The Boolean values in state must be converted to the submitted values of yes and no.
+	const uploadPreferencesForm: CSVUploadPreferencesForm = {
+		...uploadPreferences,
+		gzip: uploadPreferences.gzip ? BooleanTypes.true : BooleanTypes.false,
+		headerRow: uploadPreferences.headerRow ? BooleanTypes.true : BooleanTypes.false,
+		update: uploadPreferences.update ? BooleanTypes.true : BooleanTypes.false
+	};
+	for (const [preference, value] of Object.entries(uploadPreferencesForm)) {
+		formData.append(preference, value.toString());
 	}
+	formData.append('csvfile', metersFile); // It is important for the server that the file is attached last.
 
-	public async submitMeters(uploadPreferences: MetersCSVUploadPreferencesItem, metersFile: File): Promise<void> {
-		const formData = new FormData();
-		// The Boolean values in state must be converted to the submitted values of yes and no.
-		const uploadPreferencesForm: CSVUploadPreferencesForm = {
-			...uploadPreferences,
-			gzip: uploadPreferences.gzip ? BooleanTypes.true : BooleanTypes.false,
-			headerRow: uploadPreferences.headerRow ? BooleanTypes.true : BooleanTypes.false,
-			update: uploadPreferences.update ? BooleanTypes.true : BooleanTypes.false
-		};
-		for (const [preference, value] of Object.entries(uploadPreferencesForm)) {
-			formData.append(preference, value.toString());
-		}
-		formData.append('csvfile', metersFile); // It is important for the server that the file is attached last.
-		await this.backend.doPostRequest<void>('/api/csv/meters', formData);
+	try {
+		const response = await backend.doPostRequest<string>('/api/csv/meters', formData);
+		// Meter Data was sent to the DB, invalidate meters for now
+		dispatch(baseApi.util.invalidateTags(['MeterData']));
+		// meters were invalidated so all meter changes will now reflect in Redux state, now return
+		return { status: true, message: response };
```

Comment:
Given this is used as a boolean where true is success, I'm curious about changing the name. Could you help me understand?